debug-scripts/packages

```diff
@@ -2,5 +2,6 @@
 set -ux
 
 dpkg --list > $DEBUG_SCRIPT_DIR/dpkg-list
+snap list > $DEBUG_SCRIPT_DIR/snap-list
```

Comment:
I have no idea. The upside is it won't bomb the script if it fails; we'll just be missing that bit of info.